GameOfLife/GameOfLife/Iteration.cs

```diff
@@ -0,0 +1,162 @@
+﻿namespace GameOfLife
+{
+    public class Iteration
+    {
+        public List<int> cellsToDieX = new List<int>();
```

Comment:
There is such structure in c# called Tuple you can use that to combine cellsToDieX and cellsToDieY into one List with X and Y coordinates.

Reply:
Combined these list into corresponding lists of Tuple.

Reply:
You can mark this comment as resolved.
For some reason can't do it myself.